.github/workflows/dlp-pipelines.yml

```diff
@@ -29,7 +30,29 @@ env:
   REIDENTIFICATION_QUERY_FILE: "reid_query.sql"
 
 jobs:
+
+  build-and-unit-test:
+    permissions:
```

Comment:
Please add line breaks wherever required for better readability and consistency.

Reply:
it's actually other way around - we should remove excess new lines from whole file and use consistent yaml formatter to adhere to syntax - please see: https://g3doc.corp.google.com/devtools/yaml_format/README.md?cl=head which was used to format this section.

I'll update rest of the file to adhere to this in subsequent PR #167